run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -f ./ircd.yaml ]; then
+  cp oragono.yaml ircd.yaml
+fi
+
+if [ ! -f ircd.db ]; then
+  ./oragono initdb
+fi
+
+if [ ! -f tls.crt ]; then
+  ./oragono mkcerts
+fi
+
+./oragono run
```

Comment:
Does oragono daemonize itself for some reason? If so, that would explain this a bit.

Also, don't use `docker create` + `docker start`, just use `docker run`. Using `docker create` isn't totally supported.